.travis.yml

```diff
@@ -52,6 +52,7 @@ before_install:
   - git clone https://github.com/informatics-isi-edu/ermrestjs.git
   - cd ermrestjs
   - sudo npm install
+  - bower install
```

Comment:
What does this have to do with the rest of this PR?  Was it committed by accident?